backend/curriculum_tracking/api_views.py

```diff
@@ -559,6 +559,45 @@ def cancel_workshop_attendance(self, request, pk=None):
         card.delete_workshop_attendance()
         return Response(serializers.AgileCardSerializer(card).data)
 
+
+
+    ##**
+    @action(
+        detail=True,
+        methods=["GET"],  # We only want to get information so that's why this is no GET and no longer POST
+        
+        serializer_class=serializers.AgileCardSerializer,
+        #serializer_class=serializers.NegReviewSerializer,  #Couldnt get dedicated serializer to work
+        #serializer_class=serializers.NoArgs,    # Can work with no serializer, then only integer value appears when action is called
+        permission_classes=[
+            HasObjectPermission(
+                permissions=Team.PERMISSION_MANAGE_CARDS,
+                get_object=_get_teams_from_card,
+            )
+        ],
+    )
+    def get_total_nyc_and_rf_reviews(self, request, pk=None):
+        
+        #***do I need to get the serializer and check if valid first?
+        #serializer = self.get_serializer(data=request.data)
+        #if serializer.is_valid():
+            #card = self.get_object()
+
+            #return Response(serializers.AgileCardSerializer(card).data)
+        #else:
+            #return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
+        
+
+        #***below if done without using a serializer
+        #negative_reviews = models.RecruitProjectReview.objects.filter(status__in=['NYC', 'R']).filter(recruit_project__agile_card=card_object).count()
+
+        #return Response(negative_reviews)    # Notice my response is no longer serialized so therefore I don't need to worry about any serializers anymore
+
+        card_object = self.get_object() 
+        return Response(serializers.AgileCardSerializer(card_object).data)
+    
+
+
```

Comment:
Ok, so there is a lot going on here which is ok considering you are trying to explain your way of thinking.  However, I think it would be better/easier for whoever is reviewing your work if you rather write block comments and keep inline comments to a minimum.  Also, it would probably be a good idea to get rid of all the commented out code, if you think you might need it later then rather paste it inside some readme file or some personal file on your computer.  So in general, just a bit of cleaning up here.

Then also, like we discussed, we don't know yet what exactly constitutes a 'negative' review, so depending on the feedback we get from Sheena your code might change drastically.